testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCreateTableCompatibility.java

```diff
@@ -49,11 +61,19 @@ public void testDatabricksCanReadInitialCreateTable()
         try {
             assertThat(onDelta().executeQuery("SHOW TABLES FROM default LIKE '" + tableName + "'")).contains(row("default", tableName, false));
             assertThat(onDelta().executeQuery("SELECT count(*) FROM default." + tableName)).contains(row(0));
+            String showCreateTableTemplate;
+            if (databricksRuntimeVersion.equals(DATABRICKS_104_RUNTIME_VERSION)) {
```

Comment:
Apply the format within the branches of `if`-`else`.
This will spare in `getDatabricksDefaultTableProperties` the check:
```
if (databricksRuntimeVersion.equals(DATABRICKS_104_RUNTIME_VERSION)) {
```